### PR TITLE
Update saitypes.h

### PR DESCRIPTION
**Pull request by tushar-ty**
Modified bulk operation type to error mode type.

### CODE REVIEW

**Comment by kcudnik** [comment]
@lguohan just merged my PR, can you please update your PR again?

**Comment by kcudnik** [comment]
test this please

**Comment by lguohan** [comment]
dell says they want to have a look at this PR.

**Comment by tushar-ty** [comment]
If Dell has reviewed this then pls check it in.

**Comment by lguohan** [comment]
@AshokDaparthi , please review

File: inc/saitypes.h [hunk 1]
```diff
@@ -599,20 +599,20 @@ typedef struct _sai_attribute_t
...
-    SAI_BULK_OP_TYPE_STOP_ON_ERROR,
+    SAI_BULK_OP_ERROR_MODE_STOP_ON_ERROR,
 
     /**
-     * @brief Bulk operation ignores the failures and continues to create other objects
+     * @brief Bulk operation error handling mode where operation ignores the failures and continues to create other objects
      */
-    SAI_BULK_OP_TYPE_INGORE_ERROR,
-} sai_bulk_op_type_t;
+    SAI_BULK_OP_ERROR_MODE_INGORE_ERROR,
```
Comment (kcudnik) [inline inc/saitypes.h]: INGORE -> IGNORE

**Pull request merged**